ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -41,7 +42,7 @@
 
 namespace ArchiSteamFarm {
 	public sealed class ArchiWebHandler : IDisposable {
-		private static readonly Dictionary<uint, byte> CachedCardCountsForGame = new Dictionary<uint, byte>();
+		private static readonly ConcurrentDictionary<uint, byte> CachedCardCountsForGame = new ConcurrentDictionary<uint, byte>();
```

Comment:
most trading card bot get cardset info from [need referer steam.tools](https://cdn.steam.tools/data/set_data.json)

Reply:
I don't think that relying on third-party APIs is a good option here, because these data are available from Steam directly, so if third-party API breaks, we still would be able to get card counts.

Reply:
I'm with @Vital7 here. Relying on third party API to do something we can do ourselves very easily is not a good way IMHO.